tests/testDipoleFitter.py

```diff
@@ -56,86 +55,74 @@ class DipoleFitTestGlobalParams(object):
     @var offsets: pixel coordinates between lobes of dipoles
     """
 
-    def __init__(self):
+    def __init__(self, xc=np.array([65.3, 24.2]), yc=np.array([38.6, 78.5])):
         """Initialize the parameters."""
```

Comment:
You should define xc/yc in the method docstring. Also, making np arrays as default parameters makes me worried (a la http://effbot.org/zone/default-values.htm) but I don't think it's technically wrong.